src/DnDList.elm

```diff
@@ -539,6 +559,19 @@ commands stepMsg (Model model) =
             Cmd.batch
                 [ dragElementCommands stepMsg state
                 , dropElementCommands stepMsg state
+                , releasePointerCaptureCommand stepMsg state releasePointerCapture
+                ]
+
+releasePointerCaptureCommand : (Msg -> msg) -> State -> (Json.Decode.Value -> Cmd msg) -> Cmd msg
+releasePointerCaptureCommand stepMsg state releasePointerCapture =
+    case state.pointerCaptureEvent of
+        Nothing ->
+            Cmd.none
+
+        Just pointerCaptureEvent ->
+            Cmd.batch
+                [ Task.perform (stepMsg << (\_ -> ResetPointerCaptureEvent)) (Process.sleep 1)
```

Comment:
I don't think Task is required here, but I couldn't get it to compile any other way.